static/js/formatters-internal.js

```diff
@@ -550,10 +551,11 @@ export function highlightField(fieldValue, matchedSubstrings = []) {
     const { offset, length } = match;
     highlightedString = 
       highlightedString.substr(0, offset) +
-      `<mark>${fieldValue.substr(offset, length)}</mark>`+
+      `|mark|${fieldValue.substr(offset, length)}|/mark|`+
```

Comment:
doesn't this run into issues if somebody has `|mark|` in their actual text?

Reply:
yea decided to just escape in the substrings